### ISSUE TITLE
loop over model configurations using bash rather than python

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#17]
right now for the hyperparameters I noticed you have two separate arguments to either deal with the case where a single value is passed in or multiple values are (and separated by commas). it may be easier to have a single argument that takes in a single hyper-parameter value, and loop over the configurations through a bash script, especially since it seems fairly difficult to accommodate different ranges of hyperparameters in your current set-up. you could also apply this principle to the other arguments such as the model or loss type or dataset as well.

here's a minimal example of the kind of bash script that captures what I mean (note that bash is spacing sensitive so everything needs to match the format as below!), but you would include additional for loops for the various configurations that change as well as passing them as additional arguments when calling your python script. so this means that each time the python script gets called only a single value for each aspect of the configuration will be passed in at a time and you won't need to worry about parsing out strings of comma-separated values within python!

```bash
#!/bin/bash
models=('resnet18' 'vgg11')
lr_assoc_vals=(0.03 0.04 0.05)

for model in "${models[@]}"
do
    for lr_assoc in "${lr_assoc_vals[@]}"
    do
        python alcove.py --model $model --lr_assoc $lr_assoc
    done
done

```
